synapseAnnotations/data/experimentalData.json

```diff
@@ -23,9 +23,14 @@
         "source": "http://purl.obolibrary.org/obo/NCIT_C17731"
       },
       {
-        "value": "immunofluorescence",
-        "description": "An immunological procedure in which the antibodies are coupled with molecules which fluoresce under ultra violet (UV) light. This makes them particularly suitable for detection of specific antigens in tissues or on cells.",
-        "source": "http://purl.obolibrary.org/obo/NCIT_C17370"
+        "value": "Immunocytochemistry",
```

Comment:
@amapeters did you mean to remove immunofluorescence and add Immunocytochemistry here?

Reply:
No, I just meant to add autoradiography

Reply:
I can fix it real quick